libs/features/mas/docs/ccd.html

```diff
@@ -50,9 +50,17 @@
       document.head.appendChild(masCommerceService);
     </script>
 </head>
-
-<body class="spectrum spectrum--medium spectrum--light">
-  <main>
+  <body class="spectrum spectrum--medium spectrum--light">
+    <div class="sidenav">
```

Comment:
nit: you could use `aside` for element

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside

Reply:
sure :)